components/CreateLeadsButton/CreateLeadsButton.tsx

```diff
@@ -1,6 +1,5 @@
-import Image from 'next/image';
 import plusImage from '@/assets/images/leadslist-icons/add-circle.png';
-import React from 'react';
+import Image from 'next/image';
 
```

Comment:
Change this file name to `index.tsx` and also change the folder name to `create-leads-button`